### PR TITLE
refactor: change package name (again)

### PR DESCRIPTION
**Pull request by frederickfogerty**
ssia

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,22 +53,22 @@
 
 Below are some other articles that help explain responsive imagery, and how it can work alongside imgix:
 
-- [Using imgix with `<picture>`](https://docs.imgix.com/tutorials/using-imgixPicture-element). Discusses the differences between art direction and resolution switching, and provides examples of how to accomplish art direction with imgix.
+- [Using imgix with `<picture>`](https://docs.imgix.com/tutorials/usiangularPicture-element). Discusses the differences between art direction and resolution switching, and provides examples of how to accomplish art direction with imgix.
```
Comment (sherwinski) [inline README.md:56]:
```suggestion
- [Using imgix with `<picture>`](https://docs.imgix.com/tutorials/using-imgix-picture-element). Discusses the differences between art direction and resolution switching, and provides examples of how to accomplish art direction with imgix.
```

File: README.md [hunk 2]
```diff
@@ -342,7 +342,7 @@ If you'd still like to use an event listener, we recommend using [lazysizes](htt
 
 With the picture element, images can be directed to have different crops and sizes based on the browser dimensions, or any media query.
 
-It is recommended to check out our [introduction blog post about how to best use picture and the imgix API](https://docs.imgix.com/tutorials/using-imgixPicture-element) for some great tips and examples!
+It is recommended to check out our [introduction blog post about how to best use picture and the imgix API](https://docs.imgix.com/tutorials/usiangularPicture-element) for some great tips and examples!
```
Comment (sherwinski) [inline README.md:345]: Aside: this blog post gets linked twice in the README
```suggestion
It is recommended to check out our [introduction blog post about how to best use picture and the imgix API](https://docs.imgix.com/tutorials/using-imgix-picture-element) for some great tips and examples!
```

File: projects/imgix-angular/src/public-api.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/*
+ * Public API Surface of imgix-angular
+ */
+
+export * from './lib/imgix-angular.module';
+export * from './lib/ix-img.component';
+export * from './lib/ix-source.component';
```
Comment (sherwinski) [inline projects/imgix-angular/src/public-api.ts:7]: Should this also export `./lib/ix-picture.component.ts`?

**Review by sherwinski — CHANGES_REQUESTED** [review]
LGTM, and just to be clear on what's changing:
- repo: `imgix/ng` &#8594; `imgix/angular`
- package: `@imgix/ng` &#8594; `@imgix/angular`
- project: `imgix-ng` &#8594; `imgix-angular`
- module: `NgImgixModule` &#8594; `ImgixAngularModule`

**Comment by sherwinski** [comment]
oof I think I meant to add these review comments to #103 but ¯\\_(ツ)_/¯

**Comment by frederickfogerty** [comment]
Yep you're right with the naming changes @sherwinski

File: README.md (hunk 1 above)
Comment (frederickfogerty) [reply README.md:56]: Haha I was not expecting this!

File: projects/imgix-angular/src/public-api.ts (hunk 3 above)
Comment (frederickfogerty) [reply projects/imgix-angular/src/public-api.ts:7]: Sure, why not 🤷

**Pull request merged**